### DISCUSSION TITLE
Displaying Chinese characters

### DISCUSSION THREAD

**Discussion by joelreymont** [Vector35/binaryninja-api#5949] [Q&A] [answered]
**Version and Platform (required):**
- Binary Ninja Version: 4.2.6103-dev, 3db50f2a
- OS: macos
- OS Version: 15.0
- CPU Architecture: arm64

BN shows:

00137530  char data_137530[0x31] = "\xe5\xbd\x93\xe5\x89\x8d\xe6\x8e\xa5\xe6\x94\xb6\xe6\x9c\xba\xe7\x9a\x84\xe7\xb1\xbb\xe5\x9e\x8b\xe4\xb8\x8d\xe6\x94\xaf\xe6\x8c\x81\xe5"
00137530      "\x9b\xba\xe4\xbb\xb6\xe5\x8d\x87\xe7\xba\xa7\xef\xbc\x81", 0

but in IDA I have:

0000000137530 asc_137530      db '当前接收机的类型不支持固件升级！',0 ; DATA XREF: FwDoUpgrade+98↑o

Is there a way for BN to display Chinese characters like IDA does above?

**Comment by psifertex** [comment, marked as answer]
https://docs.binary.ninja/dev/concepts.html?h=unicode#unicode-support explains how to add support for additional Unicode code pages.  

Also, for questions instead of technical issue, please use a GitHub Discussion instead.
